src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentMethodRepository.php

```diff
@@ -26,4 +28,18 @@ public function createListQueryBuilder()
             ->leftJoin('o.translations', 'translation')
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function findEnabledForChannel(ChannelInterface $channel)
```

Comment:
Of course the typehint is oversimplified now, it's either `ArrayCollection` (if created manually) or `PersistentCollection` (if hydrated by Doctrine), but both of them implements `Collection` and `Selectable`.